frontend/src/components/studies/ReferenceGenomeSelect.tsx

```diff
@@ -1,48 +0,0 @@
-import React from 'react'
```

Comment:
Sure you don't want to use this component?

Reply:
It does not really match the rest of the form and it's missing the search. We can refactor this later if need be ... (would have taken me a lot longer to add the missing stuff to your component than matching the rest of the form pattern).